packages/core/src/types.ts

```diff
@@ -1086,7 +1097,8 @@ export enum ActionTypes {
   ErrorPlatform = 'error.platform',
   ErrorCustom = 'xstate.error',
   Pure = 'xstate.pure',
-  Choose = 'xstate.choose'
+  Choose = 'xstate.choose',
+  ActionGroup = 'xstate.group'
```

Comment:
I don't really understand the difference between `ActionTypes` and `actionTypes`. Is this the right place to add this?

Reply:
I'll leave it alone for now and if it turns out it's in the wrong spot, we can move it!